packages/api/README.md

```diff
@@ -4,14 +4,14 @@ REST API for deploying SPAs to the SPAship platform.
 
 ## Global install
 
-```
+```bash
 npm install -g @spaship/api
 spaship-api
 ```
 
 ## Hacking
 
-```
+```bash
 git@github.com:spaship/api.git
```

Comment:
This line should be correct as well

Reply:
@kunyan I'm a bit confused here since it already has the code fence tagged as bash